### ISSUE TITLE
new ship data

### ISSUE THREAD

**Issue by Yoshi-E** [MustangHX/ship_ap_calculator#21]
How do you generate the new ship data? Do you fetch the data from the game client?

**Comment by KwirkyJ** [comment]
Agreed that having a note somewhere in the source explaining how you retrieve your data (beyond "...based on US NAVY penetration formula and game file data mining." as seen in the interface) would be prudent and welcomed.
